set.mm

```diff
@@ -177619,10 +177619,10 @@ seq m ( + , ( n e. ZZ |-> if ( n e. A , [_ n / k ]_ B , 0 ) ) ) ~~> x ) \/
        4-Nov-2005.)  (Revised by Mario Carneiro, 21-Apr-2014.) $)
```

Comment:
"Proof shortened" tag?

Reply:
While changing pncan to pncand is not automatic with current tools, it is the next closeest thing. I didn't feel like it needed an OLD version so that translated to not adding a tag too. But it can be added if wanted

Reply:
An OLD version is not necessary, but a tag would be nice.